docs/production/postgresql.md

```diff
@@ -1,15 +1,15 @@
 PostgreSQL database details
 =========================
 
-Starting with Zulip 3.0, Zulip supports using PostgreSQL 10, 11, or 12,
-defaulting to PostgreSQL 12 for new installations.
+Starting with Zulip 3.0, Zulip supports using PostgreSQL 10, 11, 12,
+or 13, defaulting to PostgreSQL 13 for new installations.
```

Comment:
This sentence needs a tweak since Zulip 3.0 doesn't support 13; maybe this:
"""
Since Zulip 3.0, Zulip has supported a range of PostgresQL versions.  PostgreSQL
13 is the current default for new installations, with 10, 11, and 12 also supported.
"""

Reply:
Good catch, thanks!  Tweaked this wording.